packages/office-addin-lint/src/commands.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as commander from "commander";
+import {logErrorMessage, parseNumber} from "office-addin-cli";
+import * as defaults from "./defaults";
+import * as prettierLint from "./lint";
+
+/**
+ * Determines path to files to run lint against. Priority order follows:
+ * 1 --files option passed through cli
+ * 2 lint_files config property in package.json
+ * 3 default location
+ * @param command command options which can contain files
+ */
+function getPathToFiles(command: commander.Command):string {
+    const pathToFiles: any = command.files ? command.files : process.env.npm_package_config_lint_files;
```

Comment:
I'm not sure if it will pick up the config from the correct package.json. It should come from the package where this is being used, not this package.json itself.